test/testenv/rancher.go

```diff
@@ -196,6 +195,9 @@ func DeployRancher(ctx context.Context, input DeployRancherInput) {
 	if input.RancherImageTag != "" {
 		values["rancherImageTag"] = input.RancherImageTag
 	}
+	if input.RancherIngressClassName != "" {
+		values["ingress.ingressClassName"] = input.RancherIngressClassName
```

Comment:
Shouldn't rancher pod use the only ingress class available in the cluster by default? We install ngrok or nginx but never both in different scenarios, and I think this `RancherIngressClassName` is not needed.